languages/translations/en.json

```diff
@@ -238,37 +238,51 @@
             "default": {
                 "default": "Take the ramp",
                 "name": "Take the ramp onto {way_name}",
-                "destination": "Take the ramp towards {destination}"
+                "destination": "Take the ramp towards {destination}",
+                "exit": "Take exit {exit}",
+                "exit_destination": "Take exit {exit} towards {destination}"
```

Comment:
Wasn't sure about using an `_` here style wise. There isn't a precedent for complex/multiple fields.